internal/config.go

```diff
@@ -146,7 +150,10 @@ func NewConfigFromFile(cfgPath string) (Config, error) {
 	defer configFile.Close()
 
 	var cfg Config
-	bytes, _ := ioutil.ReadAll(configFile)
+	bytes, err := io.ReadAll(configFile)
```

Comment:
Good catch! But I'd prefer if you can open a smaller pull request with just that fix, and ideally accompanied with a test, please! 🙇🏻